jcli/src/jcli_app/key.rs

```diff
@@ -99,6 +110,40 @@ pub struct ToPublic {
     output_file: OutputFile,
 }
 
+#[derive(StructOpt, Debug)]
+pub struct Sign {
+    /// path to file with bech32-encoded secret key
+    ///
+    /// supported key formats are: ed25519, ed25519bip32, ed25519extended and sumed25519_12
+    #[structopt(long = "secret-key", short = "s")]
+    secret_key: PathBuf,
+
+    /// path to file with data to sign, if no value is passed, standard input will be used
+    #[structopt(long = "data", short = "d")]
```

Comment:
actually, now thinking about it. ideally the command would be as follow:

```
jcli key sign --secret-key my_secret_key.key contract.pdf
```

and 

```
jcli key verify --public-key pub.key --signature contact.signature contract.pdf
```

i.e. having `data` as a  positioned argument.